README.md

```diff
@@ -1,7 +1,44 @@
 # pyCascadia
 Implementation of GEBCO cookbook remove-restore and other cleaning of topography/bathymetry. Uses `pyGMT`.
 
-## Shape file closing
+## Installation
+Some dependencies of the package are a bit fiddly to install, especially on Windows.
+We recommend to use a conda environment, and install the more difficult dependencies through `conda-forge`,
+by running the commands below in sequence:
+1. Clone this GitHub repository.
```

Comment:
Users may not be used to github

````suggestion
1. Clone this GitHub repository.
```
git clone https://github.com/UCL/pyCascadia.git
```
````